.github/CODEOWNERS

```
@@ -1887,6 +1918,7 @@ x-pack/plugins/security_solution/server/lib/security_integrations @elastic/secur
 
 # Ent. Search design
 /x-pack/plugins/enterprise_search/**/*.scss @elastic/search-design
+/x-pack/test/accessibility/apps/group3/enterprise_search.ts @elastic/search-design
```

These should be owned by `search-kibana` instead of design IMO
```suggestion
/x-pack/test/accessibility/apps/group3/enterprise_search.ts @elastic/search-kibana
```